velox/functions/lib/string/StringImpl.h

```diff
@@ -566,6 +566,39 @@ FOLLY_ALWAYS_INLINE bool splitPart(
   return false;
 }
 
+template <
+    bool leftTrim,
+    bool rightTrim,
+    typename TOutString,
+    typename TInString>
+FOLLY_ALWAYS_INLINE void trimAsciiSpace(
```

Comment:
It is confusing to have trimAsciiSpace and trimAsciiWhiteSpace as they sound the same, but are different. Consider adding template parameter to trimAsciiWhiteSpace to specify the lambda for checking whether a character is white space or not.